develop/parachain-devs/runtime-development/FRAME/add-pallet-to-the-runtime.md

```diff
@@ -0,0 +1,143 @@
+
+The purpose of this article is to help you:
+
+- Learn how to update runtime dependencies to integrate a new pallet
+- Understand how to configure pallet-specific Rust traits to enable the pallet's functionality
+- Grasp the entire workflow of integrating a new pallet into your runtime
+
+## Configuring Runtime Dependencies
+
+For Rust programs, this configuration is defined in the `Cargo.toml` file, which specifies the settings and dependencies that control what gets compiled into the final binary. Since the Polkadot SDK runtime compiles to both a native binary (which includes standard Rust library functions) and a Wasm binary (which does not include the standard Rust library), the `runtime/Cargo.toml` file manages two key aspects:
+
+- The locations and versions of the pallets that are to be imported as dependencies for the runtime
+- The features in each pallet that should be enabled when compiling the native Rust binary. By enabling the standard (`std`) feature set from each pallet, you ensure that the runtime includes the functions, types, and primitives necessary for the native build, which are otherwise excluded when compiling the Wasm binary
+
+!!! note
+    For information about adding dependencies in `Cargo.toml` files, see [Dependencies](https://doc.rust-lang.org/cargo/guide/dependencies.html){target=\_blank} page in the Cargo documentation. For information about enabling and managing features from dependent packages, see [Features](https://doc.rust-lang.org/cargo/reference/features.html){target=\_blank} section in the Cargo documentation.
+
+## Dependencies for a New Pallet
+
+To add the dependencies for a new pallet to the runtime, you need to modify the `Cargo.toml` file by adding a new line into the `[workspace.dependencies]` section with the pallet you want to add. This pallet definition might look like:
+
+```toml
+pallet-example = { version = "4.0.0-dev", default-features = false }
+```
+
+This line imports the pallet-example crate as a dependency and specifies the following:
+
+- `version` - the specific version of the crate to import
+- `default-features` - determines the behavior for including pallet features when compiling the runtime with standard Rust libraries
+
+!!! note
+    If you’re importing a pallet that isn’t available on [`crate.io`](https://crates.io/){target=\_blank}, you can specify the location of the pallet (either locally or from a remote repository) by using the `git` or `path` key. For example:
+
+    ```toml
+    pallet-example = { 
+        version = "4.0.0-dev",
+        default-features = false,
+        git = "INSERT_PALLET_REMOTE_URL",
+    }
+    ```
+
+    In this case, replace `INSERT_PALLET_REMOTE_URL` with the correct repository URL. For local paths, use the path key like so:
+
+    ```toml
+    pallet-example = { 
+        version = "4.0.0-dev",
+        default-features = false,
+        path = "relative/path/to/pallet",
```

Comment:
Probably we should use the same syntax as above like 'INSERT_PALLET_RELATIVE_PATH'